### PR TITLE
add button hover customize

### PR DESCRIPTION
**Pull request by Immdeviserov**
Hi! It is awesome tool and i realize i want to improve it.
I just want to add next feature: customize hover effect on button.
In this pr i tried to do this

Maybe need some design fixes (why labels have margin-top btw?) and i dont sure about transition property

### CODE REVIEW

File: src/components/AppMasthead.vue [hunk 1]
```diff
@@ -11,10 +11,15 @@
...
+      ref="button"
       :style="
-        `borderRadius: ${leftoptions.buttonRadius}px; background: ${rightoptions.buttonColor}`
-      "
+        `borderRadius: ${leftoptions.buttonRadius}px;
+         background: ${rightoptions.buttonColor};
+         transition: background linear .1s;
+        `"
       v-if="rightoptions.button"
+      @mouseover="buttonMouseOver"
```
[the remark this answers is not in the document]
Comment (Immdeviserov) [reply src/components/AppMasthead.vue:21]: Sorry, did i understand you correctly? How can we use :hover pseudo-class in style bindings? I use mouseover and mouseout listeners just becouse i dont know how to do this

File: src/components/AppCodeOutput.vue [hunk 2]
```diff
@@ -77,6 +77,7 @@ h1 {
       return `
 button {
   background: ${this.rightoptions.buttonColor};
+  transition: background linear .1s;
```
[the remark this answers is not in the document]
Comment (Immdeviserov) [reply src/components/AppCodeOutput.vue:80]: Ok, seems cool!

**Comment by Immdeviserov** [comment]
Added 2 from 3 requested changes, and i am waitng for answer about the last one

**Comment by sdras** [comment]
Ah yeah, sorry, was thinking it could be inline without a method but that's very long and you're using refs, it's cool! Thanks

**Comment by sdras** [comment]
Oh no, looks like you accidentally changed all button colors to the hover color. Can you please revert that? The button color should remain the old color, and the hover color should be the new lighter color. They shouldn't be the same, that would make the functionality redundant.

**Comment by Immdeviserov** [comment]
I felt that i am doing something wrong :D It is just a misunderstanding, will fix in soon

**Comment by sdras** [comment]
No worries, thanks!

**Comment by Immdeviserov** [comment]
I have made the button color great again

**Comment by sdras** [comment]
thanks!

**Pull request merged**